packages/lib/models/Note.test.ts

```diff
@@ -332,4 +333,49 @@ describe('models_Note', function() {
 		expect(sortedNotes3[4].id).toBe(note2.id);
 	}));
 
+	it('should copy conflicted note to target folder and cancel conflict', (async () => {
+		// Prepare
+		const srcfolder = await Folder.save({ title: 'Source Folder' });
+		const targetfolder = await Folder.save({ title: 'Target Folder' });
+
+		const note1 = await Note.save({ title: 'note', parent_id: srcfolder.id });
+
+		const conflictedNote = await Note.createConflictNote(note1, ItemChange.SOURCE_SYNC);
+
+		// COPY File
```

Comment:
Your comment explains nothing and is even confusing since we aren't moving files but notes, so please remove it.

Reply:
Noted. Also no idea myself where did "File" come from 😂 sorry about that